test/framework/src/main/java/org/elasticsearch/test/MockLogAppender.java

```diff
@@ -241,8 +241,17 @@ public String toString() {
     }
 
     public Releasable capturing(Class<?>... classes) {
-        start();
         final var loggers = Arrays.stream(classes).map(LogManager::getLogger).toArray(Logger[]::new);
+        return appendToLoggers(loggers);
+    }
+
+    public Releasable capturing(String... names) {
+        final var loggers = Arrays.stream(names).map(LogManager::getLogger).toArray(Logger[]::new);
```

Comment:
nit: not sure why I used `toArray` here in the first place, `toList()` would be neater

Reply:
Yep I updated both places to use `toList`.